controllers/storagecluster/prometheus.go

```diff
@@ -42,20 +41,20 @@ func (r *StorageClusterReconciler) enablePrometheusRules(instance *ocsv1.Storage
 	return nil
 }
 
-func getPrometheusRules(isExternal bool) (*monitoringv1.PrometheusRule, error) {
+func getPrometheusRules(instance *ocsv1.StorageCluster) (*monitoringv1.PrometheusRule, error) {
 	rule := &monitoringv1.PrometheusRule{
 		TypeMeta: metav1.TypeMeta{
 			Kind:       monitoringv1.PrometheusRuleKind,
 			APIVersion: monitoringv1.SchemeGroupVersion.String(),
 		},
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      ruleName,
-			Namespace: ruleNamespace,
+			Namespace: instance.Namespace,
```

Comment:
We should use the namespace of operator here.

Reply:
using operator ns.